packages/cli/lib/services/sdk.ts

```diff
@@ -213,4 +213,45 @@ export class NangoSyncCLI extends NangoSyncBase {
 
         return super.getMetadata<TMetadata>();
     }
+
+    public override async getRecordsByIds<K = string | number, T = any>(ids: K[], model: string): Promise<Map<K, T>> {
+        const objects = new Map<K, T>();
+
+        if (ids.length === 0) {
+            return objects;
+        }
+
+        const externalIds = ids.map((id) => String(id).replaceAll('\x00', ''));
+        const externalIdMap = new Map<string, K>(ids.map((id) => [String(id), id]));
+
+        let cursor: string | null = null;
+        for (let i = 0; i < ids.length; i += 100) {
+            const batchIds = externalIds.slice(i, i + 100);
```

Comment:
why 100? should it be higher to save a few round trip?

Reply:
Sure, I was just setting it to some arbitrary value - what would you like it to be?